### ISSUE TITLE
Mali GPU Support

### ISSUE THREAD

**Issue by css459** [css459/arch-linux-arm-clockworkpi-a06#4]
GPU acceleration currently does not work even with the proper `midgard` drivers installed. The GPU appears in `lshw` and its frequency can be changed successfully using the Gearbox application (verified via device files). However, `glxinfo -B` (via `mesa-utils` and `mesa-demos`) reports that the `Device` is `llvmpipe` and `Accelerated: no`

**Comment by lonewsk** [comment]
You don't use mesa?

Output of `glxinfo -B`:
`lwsk @ /home/lwsk _ glxinfo -B
name of display: :0
display: :0  screen: 0
direct rendering: Yes
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Panfrost (0xffffffff)
    Device: Mali-T860 (Panfrost) (0xffffffff)
    Version: 21.3.6
    Accelerated: yes
    Video memory: 3864MB
    Unified memory: yes
    Preferred profile: core (0x1)
    Max core profile version: 3.1
    Max compat profile version: 3.1
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.1
OpenGL vendor string: Panfrost
OpenGL renderer string: Mali-T860 (Panfrost)
OpenGL core profile version string: 3.1 Mesa 21.3.6
OpenGL core profile shading language version string: 1.40
OpenGL core profile context flags: (none)

OpenGL version string: 3.1 Mesa 21.3.6
OpenGL shading language version string: 1.40
OpenGL context flags: (none)

OpenGL ES profile version string: OpenGL ES 3.1 Mesa 21.3.6
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.10`

**Comment by css459** [comment]
Is this what you see when using a clean install of the root fs? If so, it could be that I configured something incorrectly on my personal install, prior to checking.

**Comment by lonewsk** [comment]
I don't think so, but can't remember correctly. I guess I just installed `mesa` before using any graphical apps.

It worked right away tho.

**Comment by css459** [comment]
Alright thanks I’ll give it a try on a clean install. A06, right?

**Comment by lonewsk** [comment]
Yep, A06.
